fastai-food101/train.py

```diff
@@ -0,0 +1,26 @@
+from fastai import *
+from fastai.vision import *
+
+import wandb
+from wandb.fastai import WandbCallback
+wandb.init()
```

Comment:
Please add project and entity names to ensure reproducibility.

`wandb.init(project="fastai-food-101", entity="wandb-examples")`

Reply:
Done. I also added a section in the README.